File: fizzbuzz.c
#include <stdio.h>

int main(void) {
    int i;

    for (i = 0; i <= 100; i++) 
        if (i % 3 == 0 || i % 5 == 0) 
            printf("%s%s\n", 
                i%3==0 ? "Fizz":"", 
                i%5==0 ? "Buzz":"" 
            );
        else 
            printf("%d\n", i);

    return 0;
}

File: FizzBuzz.java
public class FizzBuzz {
    public static void main(String[] args) {
        String out;
        for (int i = 0; i<=100; i++) {
            out = "";
            if (i % 3 == 0) {
                out += "Fizz";
            }
            if (i % 5 == 0) {
                out += "Buzz";
            }
            if (out.length() == 0) {
                out = String.format("%d", i);
            }
            System.out.println(out);
        }
    }
}
